src/scripts/ci/publish.ts

```diff
@@ -318,8 +318,8 @@ async function getNewPatchBetaVersion(packages: Packages): Promise<string> {
   return `2.0.0-beta.${currentBeta}-${maxIncrement + 1}`
 }
 
-function getAlphaVersionIncrements(versions: string[]): number[] {
-  const regex = /2\.0\.0-alpha\.(\d+)/
+function getDevVersionIncrements(versions: string[]): number[] {
+  const regex = /2\.0\.0-dev\.(\d+)/
```

Comment:
This will not be `2.0.0` in the future, first dev versions will already be `2.1.0`.

Reply:
Should be `/2\.\d+\.\d+-dev\.(\d+)/` and replaced in other locations as well